optuna/visualization/_pareto_front.py

```diff
@@ -22,6 +26,15 @@
 _logger = optuna.logging.get_logger(__name__)
 
 
+class _ParetoFrontInfo(NamedTuple):
+    n_targets: int
+    target_names: Sequence[str]
+    best_trials_with_values: Sequence[Tuple[FrozenTrial, Sequence[float]]]
```

Comment:
The `best_trials_with_values` can be `None`. How about adding `Optional` to this? Then, we can remove `# type: ignore` from line 311.